storage/pebble/lookup_test.go

```diff
@@ -17,15 +17,20 @@ func Test_lookupKey_Bytes(t *testing.T) {
 	expectedHeight := uint64(777)
 	key := newLookupKey(expectedHeight, flow.RegisterID{Owner: "owner", Key: "key"})
 
```

Comment:
can we add a test for different types of prefixes?

Reply:
this should already be covered by `Get` and `LatestHeight` tests (the blank owner check returns an error as the `l` and `f` prefixes are skipped)